test_node_env/expected_writer.js

```diff
@@ -0,0 +1,18 @@
+import { Script } from 'vm';
+import { createWriteStream } from 'fs';
+
+let writer2 = createWriteStream("test_node_env/expected.txt");
+const args = process.argv.slice(2);
+const expected = args[0];
+
+writer2.once('open', function(fd) {
+    let r = e.runInNewContext();
+    if (typeof r !== 'undefined') {
+        writer2.write(JSON.stringify(r));
+    } else {
+        writer2.write("undefined");
+    }
+    writer2.end();
+});
+
+let e = new Script(expected);
```

Comment:
Let's use a more descriptive variable name here. How about expectedScript?

Reply:
Yea, it makes sense